console/v1/types_console_link.go

```diff
@@ -34,7 +35,7 @@ type ConsoleLinkSpec struct {
 
 // ApplicationMenuSpec is the specification of the desired section and icon used for the link in the application menu.
 type ApplicationMenuSpec struct {
-	// section is the section of the application menu in which the link should appear.
+	// section is the user defined section of the application menu in which the link should appear.
```

Comment:
Added `user defined` because generally, you assume these sections are well known and previously exist.  We should clarify that we will create the sections on the fly.

Reply:
I'm not sure this is the right comment. You can also add to existing sections like "Red Hat Applications". This almost sounds like you have to create the section somehow before creating the link.

I'd say something like

```suggestion
	// section is the section of the application menu in which the link should appear.
	// This can be any text that with appear as a subheading in the application menu
	// dropdown.
```

Reply:
```
// section is the section of the application menu in which the link should appear.
// This can be any text that will appear as a subheading in the application menu dropdown.
// A new section will be created if the text does not match text of an existing section.
```

Reply:
Updated.